cocli/cmd/corimDisplay.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim/corim"
+)
+
+var (
+	corimDisplayCorimFile *string
+	corimDisplayShowTags  *bool
+)
+
+var corimDisplayCmd = NewCorimDisplayCmd()
+
+func NewCorimDisplayCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "display",
+		Short: "display the content of a CoRIM as JSON",
+		Long: `display the content of a CoRIM as JSON
+
+	Display the contents of the signed CoRIM signed-corim.cbor 
+	
+	  cli corim display --file signed-corim.cbor
+
+	Display the contents of the signed CoRIM yet-another-signed-corim.cbor and
+	also unpack any embedded CoRIM and CoSWID
```

Comment:
On line 35 you mean unpack any embedded CoMID and CoSWID ?

Reply:
good catch!  fixed in 1805372